docs/eventing/samples/writing-event-source-easy-way/README.md

```diff
@@ -0,0 +1,305 @@
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+EOS
+```
+
+Check the logs of the event display service. You will see a new message is pushed every second:
+```bash
+$ kubectl logs event-display-dpplv-deployment-67c9949cf9-bvjvk -c user-container
+
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: your.event.source.type
+  source: urn:event:from:your-api/resource/123
+  id: your-event-id
+  datacontenttype: application/json
+Data,
+  {
+    "hello": "World 1"
+  }
+```
+
+If you are interested in to see what is injected into the event source, check the logs of it:
+```bash
+$ kubectl logs test-heartbeats-deployment-7575c888c7-85w5t
+
+Sink URL is http://event-display.default.svc.cluster.local
+Emitting event #1
+Emitting event #2
+Event posted successfully
+Event posted successfully
+```
+
+Please note that the example code above is using _Binary_ mode for CloudEvents.
+Simply change
+```javascript
+let binding = new v1.BinaryHTTPEmitter(config);
+```
+with
+```javascript
+let binding = new v1.StructuredHTTPEmitter(config);
+```
+to employ structured mode.
+
+However, binary mode should be used in most of the cases as:
+- It is faster in terms of serialization and deserialization
+- It works better with cloudevents-aware proxies (like Knative Channels) can simply check the header instead of parsing the payload
+
+## Making use of SinkBinding
+
+`SinkBinding` is a more powerful way of making any Kubernetes resource an event source.
+
+`ContainerSource` will create the container for your event source's image and it will be `ContainerSource` responsibility to manage the container.
+
+`SinkBinding` though, will not create any containers. It will inject the sink information to the already existing Kubernetes resources.
+This is a more flexible approach as you can use any Kubernetes `PodSpecable` as an event source, such as `Deployment`, `Job`, `Knative Service`, `DaemonSet` etc.
+
+We don't need any code changes in our source for making it work with `SinkBinding`.
+
+Create the event display as in the section before:
+```bash
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: docker.io/aliok/event_display-864884f202126ec3150c5fcef437d90c@sha256:93cb4dcda8fee80a1f68662ae6bf20301471b046ede628f3c3f94f39752fbe08
+EOS
+```
+
+Create a Kubernetes deployment that runs the event source:
+```bash
+cat <<EOS |kubectl apply -f -
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: node-heartbeats-deployment
+  labels:
+    app: node-heartbeats
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: node-heartbeats
+  template:
+    metadata:
+      labels:
+        app: node-heartbeats
+    spec:
+      containers:
+      - name: node-heartbeats
+        image: path/to/image/registry/node-knative-heartbeat-source:v1
```

Comment:
if you want you can use kone for publishing the image.

Reply:
`kone` looks good but I won't mention it in the tutorial because I don't want the dependency to it.